src/v2/wallet.ts

```diff
@@ -1358,7 +1373,7 @@ Wallet.prototype.sendMany = function(params, callback) {
       'message', 'minValue', 'maxValue', 'sequenceId',
       'lastLedgerSequence', 'ledgerSequenceDelta', 'gasPrice',
       'noSplitChange', 'unspents', 'comment', 'otp', 'changeAddress',
-      'instant', 'memo'
+      'instant', 'memo',
```

Comment:
Is this legal? Most importantly - is this moral?

Reply:
legal & normal. I'm not sure the lint rules for BitGoJS, so I'll remove just to avoid the extra diff line